root/curs/gene_page.mhtml

```diff
@@ -23,14 +23,14 @@ Choose curation type for <% $gene->display_name() %>:
 %   my $type_display_name = $annotation_type->{display_name};
 %   if ($annotation_type->{feature_type} eq 'gene') {
 %    if ($annotation_type->{category} eq 'interaction') {
-%#    <li class="annotation-type">
-%#      <annotation-quick-add annotation-type-name="<% $annotation_type->{name} %>"
-%#                            feature-type="gene" feature-id="<% $gene->gene_id() %>"
-%#                            feature-display-name="<% $gene_proxy->display_name() %>"
-%#                            link-label="<% ucfirst $type_display_name %>">
-%#      </annotation-quick-add>
-%#      <help-icon key="<% $annotation_type->{name} . '_definition' %>"></help-icon>
-%#    </li>
+    <li class="annotation-type">
+      <annotation-quick-add annotation-type-name="<% $annotation_type->{name} %>"
+                            link-label="<% ucfirst($annotation_type->{display_name}) %>"
```

Comment:
It would be good to use the feature-id and feature-taxon-id attributes on annotation-quick-add so that the gene and organisms are preset in the dialog:

```
     <li class="annotation-type">
       <annotation-quick-add annotation-type-name="<% $annotation_type->{name} %>"
                             link-label="<% ucfirst($annotation_type->{display_name}) %>"
+                            feature-id="<% $gene->gene_id() %>"
                             feature-type="<% $annotation_type->{feature_type} %>"
-                            feature-taxon-id="">
+                            feature-display-name="<% $gene_proxy->display_name() %>"
+                            feature-taxon-id="<% $gene_proxy->taxonid() %>">
       </annotation-quick-add>
       <help-icon key="<% $annotation_type->{name} . '_definition' %>"></help-icon>
     </li>
```

Reply:
Thanks. When I made this change I just copied the code from the summary page because I wasn't sure what these other attributes did or how they should be used. I'll test this change now.

Reply:
I've tested now and there's multiple problems with setting these attributes: firstly, the physical interaction annotation doesn't perform the filtering by organism role that we have in #2132, so even though the organism and the gene sets correctly, there's nothing to stop you switching to another host or pathogen. On top of that, you can't edit the gene after it's first set, so you can end up with invalid organism / gene combinations like this:

![image](https://user-images.githubusercontent.com/37659591/69717552-2f571900-1104-11ea-8d10-d69767359da7.png)

(where TRI5 is a _F. graminearum_ gene).

Secondly, I've noticed that the 'Interacting gene' field never seems to enable after I added the extra attributes. It stays blank after picking an interaction type, even if the gene / organism combination is valid:

![image](https://user-images.githubusercontent.com/37659591/69717670-64636b80-1104-11ea-995b-406bfeecd9cc.png)

I think this could be to do with the fact the gene isn't selected by the user.

Reply:
> so even though the organism and the gene sets correctly, there's nothing to stop you switching to another host or pathogen

I think that's this issue: #2128 
I plan to work on that soon.

> It stays blank after picking an interaction type

I'm off to bed now but I'll dig into the logic tomorrow to find a fix.